tests/python/accuracy/public_scope.json

```diff
@@ -292,7 +292,7 @@
             {
                 "image": "coco128/images/train2017/000000000074.jpg",
                 "reference": [
-                    "anomaly_map min:0.5944074988365173 max:1.0;pred_score:1.0;pred_label:Anomalous;pred_mask min:1 max:1;"
+                    "anomaly_map min:0.5944 max:1.0000;pred_score:1.0;pred_label:Anomalous;pred_mask min:1.0000 max:1.0000;"
```

Comment:
I expected this to happen here: https://github.com/openvinotoolkit/model_api/pull/98#discussion_r1277247488. But the PR seems to be stuck. Can you just change the precision to 3 digits, not 4, to stay consistent with the rest?

Reply:
Done